eth/gasprice/gasprice_test.go

```diff
@@ -90,6 +91,10 @@ func (b *testBackend) SubscribeChainHeadEvent(ch chan<- core.ChainHeadEvent) eve
 	return nil
 }
 
+func (b *testBackend) GetFeeConfigAt(parent *types.Header) (commontype.FeeConfig, error) {
```

Comment:
can we remove this in favor of `GetFeeConfig` only

Reply:
there is also a `getFeeConfig` in the precompile, so thought it would be cleaner with `GetFeeConfigAt` (which also indicates it changes with different headers)